### ISSUE TITLE
Advanced Medical System: Patient with no heart rate only wakes up with PAK

### ISSUE THREAD

**Issue by Macusercom** [acemod/ACE3#2785]
**ACE3 Version: 3.3.2.0 (Stable)**

**Mods:**
@CBA_A3
@ACE3

**Placed ACE3 Modules:**
ACE_moduleAdvancedMedicalSettings
ACE_moduleReviveSettings

**Description:**
If a patient is unconscious, has no open wounds, enough blood and no pain, he instantly wakes up when he has a heart rate. However, if there is no heart rate, there is no way of waking him up again except the PAK. CPR doesn't work (tried it 30 times) and epinephrine shouldn't be used.

So in the end there is no way of reviving a person with no heart rate except using the PAK. So basically you can always simply just use a PAK without even stabilising a patient and sometimes you have to use it as there is no other possibility of waking him up. Also if you only allow a medic to use it and the medic has no heart rate, there is no way of reviving anyone anymore.

My suggestion: at least make an option to add a heart rate after giving CPR like it was in CSE. PAKs should only be used when you want to fully heal a unit, but not to wake the unit up when he has no heart rate.

**Steps to reproduce:**
1) Place the advanced medical system module and enable PAK only when stable, set the medical classes and override death so that a player will never die instantly.
2) Shoot someone until he has no pulse.
3) Treat the wounds, give IVs, give morphine.
4) Now the only way to wake the patient up is a PAK. CPR or anything else won't do anything at all. He stays unconscious.

**Comment by alganthe** [comment]
This is intended behavior.

**Comment by SilentSpike** [comment]
Incorrect. PAK is required for units in revive state, which isn't the equivalent of cardiac arrest.

Anyway, medical is up for review soon. The revive state system will likely be a part of that.

**Comment by Glowbal** [comment]
Will be done after #3916 and #3917

Idea is to completely revise the revive system.

**Comment by TheMagnetar** [comment]
Fixed in rewrite
